web/rest/client/features/provider/contact/getContact.feature

```diff
@@ -0,0 +1,86 @@
+Feature: Retrieve contacts
+  In order to manage contacts
+  As a client admin
+  I need to be able to retrieve them through the API.
+
+  @createSchema
+  Scenario: Retrieve the contacts json list
+    Given I add Company Authorization header
+    When I add "Accept" header equal to "application/json"
+    And I send a "GET" request to "contacts"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/json; charset=utf-8"
+    And the JSON should be equal to:
+    """
+      [
+          {
+              "name": "Test Contact name",
+              "lastname": "Test Contact Lastname",
+              "email": "testcontact@email.com",
+              "workPhoneE164": "+34456123",
+              "mobilePhoneE164": null,
+              "otherPhone": null,
+              "id": 1
+          },
+          {
+              "name": "Alice",
+              "lastname": "Allison",
+              "email": "alice@democompany.com",
+              "workPhoneE164": null,
+              "mobilePhoneE164": null,
+              "otherPhone": "101",
+              "id": 2
+          },
+          {
+              "name": "Bob",
+              "lastname": "Bobson",
+              "email": "bob@democompany.com",
+              "workPhoneE164": null,
+              "mobilePhoneE164": null,
```

Comment:
It would be great to have a row with mobilePhoneE164

Reply:
Added a record (id:1) with work, mobile and other phone